src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Tests/Commands/CreateCohort/WhenCreateCohortRequestIsHandled.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentValidation;
+using FluentValidation.Results;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Api.Client;
+using SFA.DAS.ProviderCommitments.Application.Commands.CreateCohort;
+
+namespace SFA.DAS.ProviderCommitments.Tests.Commands.CreateCohort
+{
+    [TestFixture]
+    public class WhenCreateCohortRequestIsHandled
```

Comment:
I really like the test layout of a folder name and a When class, but this is different from the 2 other testing styles we have. Does it mean we have to have multiple Fixtures() for each When case (this may not be a bad thing)